frontend/README.md

```diff
@@ -79,21 +79,64 @@ kubectl port-forward svc/metadata-envoy-service 9090:9090
 
 ### Proxy to a real cluster
 
-This requires you already have a real KFP cluster, you can proxy requests to it.
-
-1. [Install Kubeflow Pipelines](https://www.kubeflow.org/docs/pipelines/installation/overview/) based on your use case and environment.
-1. Configure `kubectl` with access to your KFP cluster. (For GCP, follow [Access to GCP cluster](https://cloud.google.com/kubernetes-engine/docs/how-to/cluster-access-for-kubectl) guide).
-1. Use the following table to determine which script to run.
+You can proxy requests from the UI running on your host machine to an actual KFP
+deployment, running on a remote or local Kubernetes cluster. This dramatically
+improves iteration time, especially since the docker build can take 20+ minutes.
+
+KFP can be deployed in single-user or multi-user mode. Since there's a delta in
+logic between between the two modes, automated tests and manual validation
+against a single-user cluster can still fail when deployed to a multi-user
+cluster. Validating against a multi-user cluster at some point in your
+development cycle is advisable, even though it requires jumping through a few
+extra hoops.
+
+#### Single-user
+
+1. Follow the [single-user KFP installation
+   instructions](https://www.kubeflow.org/docs/components/pipelines/operator-guides/installation/).
+2. Configure `kubectl` with access to your KFP cluster. (For GCP, follow [Access
```

Comment:
It feels unnecessary to have GCP specific instructions (Every vendor flavor of kubernetes may have a different way to configure access. Should we have instructions for all of them?

Reply:
Thanks for the feedback, @anishasthana! I didn't contribute the GCP line, that was already there. I have no objections to removing it FWIW.

Reply:
I'd lean towards removing it, but will defer to @HumairAK